ci/tasks/get-pr-build-docker-image-info.yml

```diff
@@ -0,0 +1,43 @@
+---
+# This task runs after a PR end to end test job has created an image build.
+#
+# The task parses information from the source repo to produce a unique image tag name and
+# also to put the image filename into a file which can be used by an image-registry put.
+#
+# The 2 files created from this are:
+#
+# image_info/tag - Contains the unique tag to apply to the docker image when pushing to a registry
+# image_info/image_filename - The filename which the built image has been written to
+#
+# It uses a single param:
+#
+# app_name - The name of the app under test, this is used as part of the unique tag name and the filename
+
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: govukpay/concourse-runner
+inputs:
+  - name: src
+outputs:
+  - name: image_info
+params:
+  app_name:
+run:
+  path: /bin/bash
+  args:
+    - -ec
+    - |
+      mkdir -p image_info/
+
+      pr_number=$(cat src/.git/resource/pr)
+      head_sha=$(cat src/.git/resource/head_sha)
+
+      # We cannot access the concourse build number in a task but we need a unique
+      # tag for our build so lets just use a UUIDv4
```

Comment:
That's annoying - I guess we'd have to pass through `$BUILD_NAME` as an input if we wanted to do this? UUID is fine here though

Reply:
Sadly the env vars are not available in the pipeline definition either, so we would have to write them out to a file which could be consumed by load var, or the task :( It's a conscious decision on the part of the concourse designers. Someone wrote a resource which would do exactly write it out and load it into a var for you, but I didn't think it was worth bringing in another concourse resource type just to avoid using a random id.